Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -401,9 +401,7 @@ open class NavigationMapView: UIView {
             
             let metersPerPoint = Projection.metersPerPoint(for: maneuverCoordinate.latitude,
                                                            zoom: mapView.cameraState.zoom)
-            
-            // TODO: Implement ability to change `shaftLength` depending on zoom level.
-            let shaftLength = max(min(30 * metersPerPoint, 30), 10)
```

Comment:
It looks like even the previous code could vary the shaft length by zoom level as measured in physical units. However, this to-do item was instead referring to making the distance configurable by developers. #3638 is now tracking this task. (The equivalent Android code is currently treating [30&nbsp;meters](https://github.com/mapbox/mapbox-navigation-android/blob/f528c3011438ba31c3383382697129fd9f581f74/libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/arrow/RouteArrowUtils.kt#L44) as a fundamental law of nature via a constant by that name. 😉)

Reply:
Thanks for the suggestion, the TODO comment is brought back to track the shaft length customization.